pkg/minikube/bootstrapper/bsutil/testdata/v1.11/containerd-api-port.yaml

```diff
@@ -6,11 +6,11 @@ api:
   bindPort: 12345
   controlPlaneEndpoint: localhost
 kubernetesVersion: v1.11.0
-certificatesDir: /var/lib/minikube/certs
+certificatesDir: /usr/local/minikube/certs
 networking:
   serviceSubnet: 10.96.0.0/12
 etcd:
-  dataDir: /var/lib/minikube/etcd
+  dataDir: /usr/local/minikube/etcd
```

Comment:
Please keep user data in /var/lib/minikube if possible.

If not: Moving the etcd directory means we lose the previous cluster state. If we're going to move it, we'll need to have code that relocates the old directory or provides a symbolic link.